src/Components/Types/type.ts

```diff
@@ -0,0 +1,17 @@
+export interface Task {
+  id: string;
+  title: string;
+  desc: string;
+  status: TaskStatus;
+}
+
+export enum TaskStatus {
```

Comment:
The enum must contain constant.

```suggestion
export enum TaskStatus {
  TODO = 'todo',
  IN_PROGRESS = 'inProgress',
  COMPLETED= 'completed',
}
```